docs/_tutorials/getting_started.md

```diff
@@ -79,13 +81,13 @@ export SLACK_SIGNING_SECRET=<your-signing-secret>
 export SLACK_BOT_TOKEN=xoxb-<your-bot-token>
```

Comment:
Hmmm...I think it may be a bit confusing to save the bot token here then switch to an app token later

Reply:
@shaydewael When I remove the `token` field from my Bolt test app and leave only `appToken`, I get an App Initialization error, so my understanding is that you need a bot token and an app token to use Socket Mode. I'm unclear why that is, personally. 

In this case the app token becomes relevant when the user is setting up Socket Mode, so that's why I figured I'd introduce it in the later section. Open to feedback that that's confusing though. We can also guide the user at this stage in getting their app-token. It would also work.